test/scenarios-bootc/presubmits/el96-src@ai-model-serving-offline.sh

```diff
@@ -34,5 +34,5 @@ scenario_run_tests() {
     local -r full_guest_name=$(full_vm_name host1)
     run_tests host1 \
         --variable "GUEST_NAME:${full_guest_name}" \
-        suites/ai-model-serving/ai-model-serving.robot
+        suites/ai-model-serving/ai-model-serving-offline.robot
```

Comment:
Should we move this scenario of periodics if we have online test?

Reply:
I don't have a strong opinion. @ggiguash wdyt?

Reply:
@ggiguash asked me to merge this as it is, because he'll work on reorganizing all the periodics and presubmits jobs in a few days